src/main.py

```diff
@@ -0,0 +1,29 @@
+# __author__ = "Vasudev Gupta"
+# __author_email__ = "7vasudevgupta@gmail.com"
```

Comment:
Are we maintaining this throughout?

Reply:
yes. But I can remove this if you ask me to do?

Reply:
@MorganR what do you think?

Reply:
deleting this for now.